common/app/templates/inlineJS/blocking/shouldEnhance.scala.js

```diff
@@ -35,7 +46,7 @@
         return (navigator.platform === 'iPad');
     };
 
-    window.shouldEnhance = !personPrefersCore() && !isOlderIOSDevice() && !(@item.isFront && isIpad());
+    window.shouldEnhance = personDemandsFeatures() || (!personPrefersCore() && !isOlderIOSDevice() && !(@item.isFront && isIpad()));
```

Comment:
If the user demands features, `isCore` would return `false`. If the user prefers core, it would return `true`. If the user has no prefs, we return `false`?